oop_hw3_v2/train.py

```diff
@@ -0,0 +1,23 @@
+
+
+class Train:
+
+    def __init__(self):
+        self.__locomotive = 1
+        self.list_of_wagons = []
+
+    def __len__(self):
+        return len(self.list_of_wagons)
```

Comment:
When using len on a train, I want to see a list of Wagons

In this implementation i see int